.github/workflows/labeler.yml

```diff
@@ -6,6 +6,9 @@ jobs:
   triage:
     runs-on: ubuntu-20.04
     steps:
-      - uses: actions/labeler@v3
+      - uses: kachkaev/labeler@d89797c51d07680aec17049cc6790e9d323d9a93
```

Comment:
https://docs.github.com/en/actions/creating-actions/about-custom-actions#using-a-commits-sha-for-release-management